src/views/credits/credits.jsx

```diff
@@ -224,7 +214,11 @@ var Credits = React.createClass({
                 Tammy Stern,
                 Lis Sylvan,
                 Claudia Urrea,
-                Oren Zuckerman
+                Oren Zuckerman,
+                Connor Hudson,
```

Comment:
Same here. Please put in alphabetical order by last name.

Reply:
On it!